src/app/client/package.json

```diff
@@ -111,7 +111,7 @@
     "karma": "2.0.0",
     "karma-chrome-launcher": "2.2.0",
     "karma-coverage": "1.1.1",
-    "karma-coverage-istanbul-reporter": "1.2.1",
+    "karma-coverage-istanbul-reporter": "^2.1.1",
```

Comment:
Please make sure we have taken latest fetch and than pushed pacakage-loack.json on top of it.
Or else it might result in revert of all the changes we did in pacakge-lock.json

Reply:
Yes @deveshMantra , I reverted my existing package.json and package-lock.json then updated the version.